### DISCUSSION TITLE
Feature request - multi user support

### DISCUSSION THREAD

**Discussion by Martin-Fox** [bljohnsondev/mishos#3] [Ideas]
Hi there!

I think the multi-user support will be great. I'm hosting some solutions for me ands my friends, and it will be great to have our own movie/seriel watch list :)

**Comment by bljohnsondev** [comment]
This is definitely on the roadmap.  I've opened up discussions for this project so for tracking purposes I will move this there instead.

The backend code already supports multiple users.  The only things missing are a UI page for adding/removing users and the addition of user roles (admin/user).

**Comment by bljohnsondev** [comment]
Sorry for the delay.  I got sidetracked with some other functionality changes.  Multi user support has now been added in version 0.2.8.

Please note that if you do upgrade you will need to set `RUN_MIGRATION=1` to update the database schema.  The migration process will also automatically set your initial user as an admin.  Once the upgrade has been completed you can comment out the `RUN_MIGRATION` variable.

Let me know if you have any issues!

**Reply by Martin-Fox** [reply]
Hi @bljohnsondev 
Sorry for a delay, and thank you for that. It's working like a charm :)
